lib/Doctrine/Import/Builder.php

```diff
@@ -708,7 +719,15 @@ public function buildPhpDocs(array $definition)
                 $name = trim($name);
                 $fieldName = trim($fieldName);
 
-                $ret[] = '@property ' . $column['type'] . ' $' . $fieldName;
+                $type = $column['type'];
+                if ($phptype = $this->getPhpType($type)) {
```

Comment:
Consider wrapping this assignment in parentheses and testing the value of the assignment-expression for non-null values.